doc/tor.md

```diff
@@ -39,7 +39,7 @@ In a typical situation, this suffices to run behind a Tor proxy:
 
 If you configure your Tor system accordingly, it is possible to make your node also
 reachable from the Tor network. Add these lines to your /etc/tor/torrc (or equivalent
-config file):
+config file): *Needed for Tor version 0.2.7.0 and older versions of Tor only.*
```

Comment:
This doesn't make it clear *why*, and implies that there is nothing to be done otherwise. 

At least link to something that describes the automatic tor listening, and specifies how to get it working. Bitcoind needs access to the tor control port `-torcontrol=<ip>:<port>`. Then, for authentication, it needs access to tor's auth cookie file, or alternatively a control password can be specified manually using `-torpassword=<pass>`.

Reply:
I take it that it will be sufficient to refer the reader to section 3?